### PR TITLE
Update of the French version

### PR DESCRIPTION
**Pull request by bubuleur**
Hello,
Update your application in French
thank you

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -115,12 +132,21 @@
...
+    <string name="summary_checkbox_usb_bind_all">Force le pilote USB de Moonlight à prendre en charge tous les gamepads Xbox pris en charge</string>
+    <string name="title_checkbox_mouse_emulation">Emulation de la souris via le gamepad</string>
+    <string name="summary_checkbox_mouse_emulation">Appuyez longuement sur le bouton Start pour faire basculer la manette de jeu en mode souris.</string>
 
     <string name="category_on_screen_controls_settings">Paramètres des contrôles à l\'écran</string>
     <string name="title_checkbox_show_onscreen_controls">Afficher les commandes à l\'écran</string>
     <string name="summary_checkbox_show_onscreen_controls">Afficher la superposition du contrôleur virtuel sur l\'écran tactile</string>
     <string name="title_only_l3r3">Montre seulement L3 et R3</string>
     <string name="summary_only_l3r3">Cacher tout sauf L3 et R3</string>
+    <string name="title_reset_osc">Effacer la disposition des commandes à l\'écran sauvegardée</string>
```
Comment (mrb113) [inline app/src/main/res/values-fr/strings.xml]: I just noticed that the tense of the verbs is not the same throughout the file. 
(ignor**er** vs. ignor**ez**, etc.). 
Mon français n'est plus très bon... Ça fait longtemps ;) Avez vous un opinion?

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -139,4 +165,9 @@
     <string name="category_advanced_settings">Réglages avancés</string>
     <string name="title_video_format">Modifier les paramètres H.265</string>
     <string name="summary_video_format">H.265 réduit les exigences de bande passante vidéo, mais requiert un périphérique très récent.</string>
+    <string name="title_video_format">Modifier les paramètres H.265</string>
+    <string name="summary_video_format">H.265 réduit les besoins en bande passante vidéo mais nécessite un périphérique très récent</string>
```
Comment (cgutman) [inline app/src/main/res/values-fr/strings.xml]: These are duplicates of the above two items

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -3,6 +3,9 @@
     <!-- Shortcut strings -->
     <string name="scut_deleted_pc">PC supprimé</string>
     <string name="scut_not_paired">PC non appairé</string>
+    <string name="scut_pc_not_found">PC non trouvé</string>
+    <string name="scut_invalid_uuid">Le PC fourni n'est pas valide</string>
+    <string name="scut_invalid_app_id">L'application fournie n'est pas valide</string>
```
Comment (cgutman) [inline app/src/main/res/values-fr/strings.xml]: `'` characters need a `\` before them

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (bubuleur) [reply app/src/main/res/values-fr/strings.xml]: Pour effacer il faut dire mordre pour er et mordu pour é enfin moi j'ai appris comme ça
et donc ici c'est " mordre la disposition des commandes....." = Effacer 
Sinon pour Effacez c'est pour "vous" = Vous effacez
Excuse moi je le dit en français

File: app/src/main/res/values-fr/strings.xml (hunk 3 above)
Comment (bubuleur) [reply app/src/main/res/values-fr/strings.xml]: thank you for the remark I had not seen the one I change that

**Comment by bubuleur** [comment]
It is possible to put this update in French in your next update of Moonlight

**Comment by cgutman** [comment]
Merged, but there are still a few strings lacking translation.

**Comment by bubuleur** [comment]
Thank you very much, but what channel is missing? I am here for this
cordially

**Pull request merged**